MAUI/Popup/layout-customizations.md

```diff
@@ -0,0 +1,451 @@
+---
+layout: post
+title: Layout Customizations in .NET MAUI Popup control | Syncfusion
+description: Learn here all about Layout Customizations support in Syncfusion .NET MAUI Popup (SfPopup) control, its elements and more.
+platform: MAUI
+control: SfPopup
+documentation: ug
+---
+
+# Layout Customizations in MAUI Popup (SfPopup)
+
+## Popup appearance mode for footer
+
+The `SfPopup` supports two types of `SfPopup.AppearanceMode`. By default, the `PopupButtonAppearanceMode.OneButton` is set. You can change the appearance by using the `SfPopup.AppearanceMode` property.

```

Comment:
The `SfPopup` supports two types of `SfPopup.AppearanceMode.` By default, the `PopupButtonAppearanceMode.OneButton` is set. Change the appearance by using the `SfPopup.AppearanceMode` property.